opm/output/eclipse/CreateInteHead.cpp

```diff
@@ -504,8 +504,8 @@ namespace {
     {
         const int noactnod = sched[lookup_step].network().node_names().size();
         const int noactbr  = sched[lookup_step].network().NoOfBranches();
-        const int nodmax = rspec.networkDimensions().maxNONodes();
-        const int nbrmax = rspec.networkDimensions().maxNoBranches();
+        const int nodmax = std::max(rspec.networkDimensions().maxNONodes(), sched[lookup_step].network().NoOfNodes());
+        const int nbrmax = std::max(rspec.networkDimensions().maxNoBranches(), sched[lookup_step].network().NoOfBranches());
```

Comment:
As far as what I understand, with RUNSPEC, we can not specify the dimensions for the Standard Network.  We can only specify for the extended network. (it is what this PR about).  Have you checked that the items related to `nbrmax` and `nodmax` is consistent with the reference output with standard network?

Reply:
No, and if possible I would like to avoid that, hence the message in `NetworkKeywordHandlers.cpp`.

Reply:
what is the reference output with standard network for nodmax and nbrmax?